modules/base/src/components/About.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+
+  <div class="about">
+    <div class="ui basic segment">
+
+      <h3>License</h3>
+
+      <p>
+        The Portmaster is licensed under the AGPLv3.<br />
+        More attribution to other Projects and their licenses coming soon.
+      </p>
+
+      <h4>Attribution</h4>
+      <table class="ui very basic compact table">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>License</th>
+            <th>Notice</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>MaxMind GeoLite2</td>
+            <td>EULA with CC BY-SA 4.0</td>
+            <td>This product includes GeoLite2 data created by MaxMind, available from <a href="http://www.maxmind.com">http://www.maxmind.com</a>.</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h3>Core Version Info</h3>
+
+      <div v-if="$parent.versions">
+        <p style="padding: 5px;">
+          <strong>Name</strong>:         {{ $parent.versions.Core.Name }}<br/>
+          <strong>Version</strong>:      {{ $parent.versions.Core.Version }}<br/>
+          <strong>Commit</strong>:       {{ $parent.versions.Core.Commit }}<br/>
+          <strong>BuildOptions</strong>: {{ $parent.versions.Core.BuildOptions }}<br/>
+          <strong>BuildUser</strong>:    {{ $parent.versions.Core.BuildUser }}<br/>
+          <strong>BuildHost</strong>:    {{ $parent.versions.Core.BuildHost }}<br/>
+          <strong>BuildDate</strong>:    {{ $parent.versions.Core.BuildDate }}<br/>
+          <strong>BuildSource</strong>:  {{ $parent.versions.Core.BuildSource }}<br/>
```

Comment:
For better readability consider adding a space after each `Build` (i.e. `Build Options`)

Reply:
Yup. Good point.